karate-tests/src/test/java/mocks/sandbox/patch-patient.js

```diff
@@ -1,146 +1,154 @@
-function buildResponseHeaders(request, patient) {
-    return {
-        'content-type': 'application/fhir+json',
-        'etag': `W/"${patient.meta.versionId}"`,
-        'x-request-id': request.header('x-request-id'),
-        'x-correlation-id': request.header('x-correlation-id')
-    }
+/* Karate objects */
+/* global context, request, response, session */
+
+/* Functions defined in supporting-functions.js */
+/* global validateHeaders, validateNHSNumber, validatePatientExists, basicResponseHeaders */
+
+function buildResponseHeaders (request, patient) {
+  return {
+    'content-type': 'application/fhir+json',
+    etag: `W/"${patient.meta.versionId}"`,
+    'x-request-id': request.header('x-request-id'),
+    'x-correlation-id': request.header('x-correlation-id')
+  }
 }
 
 /*
     Diagnostics strings for error messages
 */
-const NO_IF_MATCH_HEADER = "Invalid update with error - If-Match header must be supplied to update this resource";
-const NO_PATCHES_PROVIDED = "Invalid update with error - No patches found";
-const INVALID_RESOURCE_ID = "Invalid update with error - This resource has changed since you last read. Please re-read and try again with the new version number.";
-const INVALID_PATCH = "Invalid patch: Operation `op` property is not one of operations defined in RFC-6902"
+const NO_IF_MATCH_HEADER = 'Invalid update with error - If-Match header must be supplied to update this resource'
+const NO_PATCHES_PROVIDED = 'Invalid update with error - No patches found'
+const INVALID_RESOURCE_ID = 'Invalid update with error - This resource has changed since you last read. Please re-read and try again with the new version number.'
+const INVALID_PATCH = 'Invalid patch: Operation `op` property is not one of operations defined in RFC-6902'
 
 /*
     Functions to handle error responses
 */
-function setInvalidUpdateError(request, diagnostics) {
-    let body = context.read('classpath:mocks/stubs/errorResponses/INVALID_UPDATE.json')
-    body.issue[0].diagnostics = diagnostics
-    response.headers = basicResponseHeaders(request)
-    response.body = body
-    response.status = 400
-    return false
+function setInvalidUpdateError (request, diagnostics) {
+  const body = context.read('classpath:mocks/stubs/errorResponses/INVALID_UPDATE.json')
+  body.issue[0].diagnostics = diagnostics
```

Comment:
`body` is defined as a const and we're changing a property within it. Should this be left as a `let` instead?

Reply:
Objects within a const can be altered. Will remove similar comments